typescript/infra/src/config/gas-oracle.ts

```diff
@@ -236,16 +236,13 @@ export function getTokenExchangeRateFromValues(
   // Workaround for chicken-egg dependency problem.
   // We need to provide some default value here to satisfy the config on initial load,
   // whilst knowing that it will get overwritten when a script actually gets run.
-  if (!tokenPrices[local] || !tokenPrices[remote]) {
-    return BigNumber.from(1);
```

Comment:
my read here is that if either price is undefined, we want to return an exchange rate of 1. The new logic seems to have a new case where e.g. `tokenPrices[local]` can be undefined and `tokenPrices[remote]` is defined, and we'd return `1 / tokenPrices[local]` with the added premium. Is this intended?

Reply:
yup this is intended. without this change I was getting underflow/overflow errors on some exchange pairings.

It's a weird one because this case of tokenPrices[X] being undefined is only on the initial config loading, which is quickly overriden when `print-token-prices` is ran. The root problem here is a dependency issue between the config/jsons/scripts that I've not had the time to fix yet

Reply:
would be useful to add a comment or TODO to clarify

Reply:
there is just above the diff https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/449945d53d10cadb84fa65604507ce9cfbdcbeb3/typescript/infra/src/config/gas-oracle.ts#L236-L238

Reply:
I'll add a comment about the under/overflow too